.github/workflows/codeql-analysis.yml

```
@@ -29,6 +29,8 @@ jobs:
         with:
           languages: ${{ matrix.language }}
       - name: Autobuild
+        env:
+          CODEQL_ACTION_RUN_MODE: action
```

```suggestion
          CODEQL_ACTION_RUN_MODE: Action
```

I'm seeing some capitalized usages of this (like [here](https://github.com/github/codeql-action/issues/912)); it can't hurt to try it that way.